Cargo.toml

```diff
@@ -2,6 +2,17 @@
 name = "remem"
 version = "0.1.0"
 authors = ["Ryan Levick <ryan.levick@gmail.com>"]
```

Comment:
@yoshuawuyts feel free to add yourself to this list.

Reply:
:tada: